libraries/rush-lib/src/logic/StandardScriptUpdater.ts

```diff
@@ -96,11 +166,24 @@ export class StandardScriptUpdater {
   private static _normalize(content: string): string {
     // Ignore newline differences from .gitattributes
     return (
-      Text.convertToLf(content)
```

Comment:
Why did you remove `convertToLf()` here?

Reply:
The `trimRight()` on each line will trim the `\r` if the file is CRLF.